packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -50,6 +49,7 @@ contract DeployImplementationsInput {
     // Outputs from DeploySuperchain.s.sol.
     SuperchainConfig internal _superchainConfigProxy;
     ProtocolVersions internal _protocolVersionsProxy;
+    ProxyAdmin internal _superchainProxyAdmin;
```

Comment:
Change made based on comment: https://github.com/ethereum-optimism/optimism/pull/11875#discussion_r1756993421

Reply:
I just realized we don't need this as an input, since we can read this from the `superchainConfigProxy` as `Proxy(superchainConfigProxy).admin()`. We'll have to prank as the zero address for that call since it's behind `proxyCallIfNotAdmin`

Reply:
@blmalone Unresolved this one, curious on your thoughts here?

Reply:
What we could do is keep the `DeployImplementationsInput.superchainProxyAdmin()` getter method, but just have it make a call to the address returned from `DeployImplementationsInput.superchainConfigProxy()` to get it's value. This minimizes refactoring since you don't need to add new getters everywhere, and the rationale is that `superchainProxyAdmin` is an "implicit input", i.e. we do need it, but we just can infer it from other inputs so it has a getter but cannot be set directly

Reply:
Makes sense, cleaner that way. Added in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/f0b349d08222c75cb44a107083c8f6361f3ff3c8